README.md

```diff
@@ -710,110 +723,111 @@ and if the data type is nullable, all when applicable.
     "type": "string",
     "format": "email",
-    "nullabe": true
-  },
-  "EmailAddressCollection": {
-    "description": "Comma separated list of email addresses defined by RFC 5322.",
-    "example": "info@qowaiv.org,test@test.com",
-    "type": "string",
-    "format": "email-collection",
-    "nullabe": true
+    "nullable": true
   },
   "HouseNumber": {
     "description": "House number notation.",
     "example": "13",
     "type": "string",
     "format": "house-number",
-    "nullabe": true
+    "nullable": true
   },
   "LocalDateTime": {
     "description": "Date-time notation as defined by RFC 3339, without time zone information.",
     "example": "2017-06-10 15:00",
     "type": "string",
     "format": "local-date-time",
-    "nullabe": false
+    "nullable": false
   },
   "Month": {
     "description": "Month(-only) notation.",
     "example": "Jun",
     "type": "string",
     "format": "month",
-    "nullabe": true,
-    "enum": ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec", "?"]
+    "nullable": true,
+   "enum": ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec", "?"]
   },
   "MonthSpan": {
     "description": "Month span, specified in years and months.",
     "example": "1Y+10M",
     "type": "string",
     "format": "month-span",
     "pattern": "[+-]?[0-9]+Y[+-][0-9]+M",
-    "nullabe": false
+    "nullable": false
   },
   "Percentage": {
     "description": "Ratio expressed as a fraction of 100 denoted using the percent sign '%'.",
     "example": "13.76%",
     "type": "string",
     "format": "percentage",
     "pattern": "-?[0-9]+(\\.[0-9]+)?%",
-    "nullabe": false
+    "nullable": false
   },
   "PostalCode": {
     "description": "Postal code notation.",
     "example": "2624DP",
     "type": "string",
     "format": "postal-code",
-    "nullabe": true
+    "nullable": true
   },
   "Sex": {
     "description": "Sex as specified by ISO/IEC 5218.",
     "example": "female",
     "type": "string",
     "format": "sex",
-    "nullabe": true,
+    "nullable": true,
     "enum": ["NotKnown", "Male", "Female", "NotApplicable"]
   },
   "Uuid": {
     "description": "Universally unique identifier, Base64 encoded.",
     "example": "lmZO_haEOTCwGsCcbIZFFg",
     "type": "string",
     "format": "uuid-base64",
-    "nullabe": true
+    "nullable": true
   },
   "WeekDate": {
     "description": "Full-date notation as defined by ISO 8601.",
     "example": "1997-W14-6",
     "type": "string",
     "format": "date-weekbased",
-    "nullabe": false
+    "nullable": false
   },
   "Year": {
     "description": "Year(-only) notation.",
     "example": 1983,
     "type": "integer",
     "format": "year",
-    "nullabe": true
+    "nullable": true
+  },
+  "YearMonth": {
+    "description": "Date notation with month precision.",
+    "example": "2017-06",
+    "type": "string",
+    "format": "year-month",
+    "pattern": "[0-9]{4}-[01][0-9]",
```

Comment:
Matches `2024-13`.
`[0-9]{4}-(0[1,9]|1[0,2])`

Reply:
Nice improvement! Obviously, this still matches `0000-12`, but as these patterns are helping to explain the data, not the actual validation (at least one should use this struct to do so).